akanda/neutron/plugins/ml2_neutron_plugin.py

```diff
@@ -0,0 +1,75 @@
+# Copyright 2014 DreamHost, LLC
+#
+# Author: DreamHost, LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may
+# not use this file except in compliance with the License. You may obtain
+# a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+
+from neutron.plugins.ml2 import plugin
+from neutron.services.l3_router import l3_router_plugin
+
+from akanda.neutron.plugins import decorators as akanda
+from akanda.neutron.plugins import floatingip
+
+akanda.monkey_patch_ipv6_generator()
+
+
+class Ml2Plugin(floatingip.ExplicitFloatingIPAllocationMixin,
+                plugin.Ml2Plugin):
+
+    _supported_extension_aliases = (
+        plugin.Ml2Plugin._supported_extension_aliases +
+        ["dhrouterstatus"]
+    )
+
+    try:
+        _supported_extension_aliases.remove('agent_scheduler')
+    except ValueError:
+        pass
```

Comment:
What's the point of suppressing this error? Should we at least add a warn/debug log line before passing?